app/src/test/kotlin/compiler/e2e/VariablePropertiesAnalysisErrorsTest.kt

```diff
@@ -2,13 +2,15 @@ package compiler.e2e
 
 import compiler.common.diagnostics.Diagnostic
 import compiler.e2e.common.E2eAsserter.assertErrorOfType
-import org.junit.Test
+import kotlin.test.Ignore
+import kotlin.test.Test
 
 class VariablePropertiesAnalysisErrorsTest {
     private fun assertAssignmentToParameterError(program: String) {
         assertErrorOfType(program, Diagnostic.VariablePropertiesError.AssignmentToFunctionParameter::class)
     }
 
+    @Ignore // FIXME: as this is an e2e test, the error is reported by the type checker, and not the variable properties analyzer
```

Comment:
If we check this in type checker, then why do we even test for that in `VariablePropertiesAnalyzer`?

Reply:
I don't know. We'll need to discuss this.
https://github.com/TCS-Compilers/compiler/issues/95